### PR TITLE
Build env - swarm

### PR DESCRIPTION
**Pull request by Bonczidai**
branch to continue the work on swarm use case for build env (#230)

Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

**Comment by Bonczidai** [comment]
the `coreos` and `cadvisor` changes are also merged into this branch ( #264  )

p.s. the actual changes can be seen by comparing the `ci-env` branch with this one or
after merging #264 the diff should automatically be much simpler

File: test/io/cloudslang/docker/swarm/test_demo_containers.sl [hunk 1]
```diff
@@ -29,30 +28,43 @@ flow:
...
   workflow:
-    - pre_clear_swarm_cluster:
-       do:
-         swarm.clear_cluster:
-            - swarm_manager_ip
-            - swarm_manager_port
-            - host
-            - port:
+    - setup_cluster:
+        do:
```
Comment (orius123) [inline test/io/cloudslang/docker/swarm/test_demo_containers.sl]: Why do you need to setup a cluster here? 
Clean machine is not enough for this test?

File: test/io/cloudslang/docker/swarm/test_get_cluster_info.sl [hunk 2]
```diff
@@ -29,30 +28,43 @@ flow:
...
   workflow:
-    - clear_swarm_cluster:
-       do:
-         swarm.clear_cluster:
-            - swarm_manager_ip
-            - swarm_manager_port
-            - host
-            - port:
+    - setup_cluster:
+        do:
```
Comment (orius123) [inline test/io/cloudslang/docker/swarm/test_get_cluster_info.sl]: also here?
> Why do you need to setup a cluster here? 
> Clean machine is not enough for this test?
Can't you leave the inputs file with 1?
Is it testing better?

File: test/io/cloudslang/docker/swarm/test_run_container_in_cluster.sl [hunk 3]
```diff
@@ -30,30 +29,43 @@ flow:
...
 
   workflow:
-    - pre_clear_swarm_cluster:
-       do:
-         swarm.clear_cluster:
-            - swarm_manager_ip
-            - swarm_manager_port
-            - host
-            - port:
+    - setup_cluster:
```
Comment (orius123) [inline test/io/cloudslang/docker/swarm/test_run_container_in_cluster.sl]: Guess what I'm going to comment here?
Right..
> also here
Why do you need to setup a cluster here? 
Clean machine is not enough for this test?
Can't you leave the inputs file with 1?
Is it testing better?

**Comment by orius123** [comment]
Ok.
So I see that your tests pass, but..
Do we really need to setup the entire env each time?
How much time does it take to set the environment up?

**Pull request merged**